### DISCUSSION TITLE
Le bonusligue à trop d'impact sur le tw !

### DISCUSSION THREAD

**Discussion by BananePlantain** [Crownicles/Crownicles#2431] [Suggestions fermées]
Les points données par le classement gloire a beaucoup beaucoup trop d'impact sur le classement score de la semaine.Quand on sais que le 1er à 4K et le 3e 2950 ca fait 1k d'écart soit une ferveur du roi/1 gros event/2 petits.Suffit de regarder Procion il domine le tw et paf 4k points gratuit qui le rend encore moins rattrapable .Ca fait combien de semaines consécutives ou il raflent le topweek et le top gloire en même temps ?
PS:je m'accuse personne de triche mais on perd le but premier du 🎗️ qui est que tout le monde peut l'avoir

**Comment by KirIcare** [comment]
Il "suffit" de gagner le top gloire xD

**Reply by BananePlantain** [reply]
bah ouais facile 💀

**Comment by BastLast** [comment]
bah faut commencer à devenir meilleur en fight làààààà

**Comment by OrionYp** [comment]
Impossible pour un "nouveau" joueur de gagner le tw à cause de ça

**Reply by BastLast** [reply]
le tw n'est pas fait pour être gagnable par un nouveau joueur tho

**Reply by BananePlantain** [reply]
ah bon ? et le sondage ou tu demandais au joueur ce qu'il pensait du tw et de son accessibilité et que tous le monde à répondu "doit etre accessible à tous avec comme seule avantage des anciens l'expérience/inventaire/argent"

**Reply by BastLast** [reply]
un bon classement dans le tw est toujours possible pour un débutant. le top 1 est quasi impossible en effet et je pense que c'est bien comem ça aussi ce sondage est vieux mtn

**Reply by BananePlantain** [reply]
refais en un ça sera pareil. Il est censé etre accessible...

**Reply by Voisin1234** [reply]
Je n'ai jamais constaté un bas niveau remporter (TW), et la  semaine dernière Evatco ( niveau 90) a largement dominé le classement, ne perdant qu'avec une différence de 47 points face à procion. Raven depuis plusieurs semaines, le suit à chaque fois, et ça s'acharner.. Ce n'est pas simplement une question de classement de gloire, mais plutôt une compréhension approfondie du jeu. Et un bon inventaire aide bien

**Reply by BastLast** [reply]
même si j'enlevais le bonus de gloire procion gagnerai quand même xD

**Reply by OrionYp** [reply]
> tho

Alors je voulais plutôt dire, quelqu'un qui commence le jeu aujourd'hui ne pourra jamais gagner le tw, même pas dans 2ans

**Reply by BananePlantain** [reply]
ouais mais voir que quelquun le gagne avec 5k d'écart avec le 2e c'est vite fait décourageant

**Reply by Carsebel** [reply]
Ça commence à faire bcp contre un seul joueur. 
Après les soupçons de certains sur le fait qu'il triche maintenant on ajoute le fait qu'il gagne le tw et top gloire. Y a perdonne d'autres à embêter ?  Les points du top gloire sont definis par le classe de ce même top. Plus tu gagnes plus tu montes. Le tw est pareil, plus tu joues bien plus t'engendre de points.

**Reply by Voisin1234** [reply]
Si cela était trop facile, ça ne serait même pas une victoire de l'avoir gagné

**Reply by Voisin1234** [reply]
Et même avant qu'il existe les points de gloire, celui-ci était très difficile. C'est sûr que ça aide toujours d'avoir 2000 de plus mais avec beaucoup de volonté et d'acharnement, tout est réalisable.
Le classement avant été dominé par Docteur, Hitori,le pourf ou je ne sais qui d'autre et c'est pas pour autant qu'ils arrivaient à gagner le TW

**Reply by BananePlantain** [reply]
@Carsebel le but n'est pas de viser qui que ce soit

**Reply by Ravenclaw-l** [reply]
Pourtant tu as bien cité Procion. Il n'a pas pris en otage le top gloire, pas mal de joueurs peuvent y arriver :) tout comme le top week.

**Reply by OrionYp** [reply]
Je pense qu'on ne devrait pas être obligé de combattre pour gagner le tw

**Reply by BananePlantain** [reply]
ouep y a plein de gens qui n'aiment pas combattre donc les points dans le classement score définitif ok mais pas semaine

**Reply by BastLast** [reply]
malheureusement pour toi c'est moi le dev mdr

**Reply by BananePlantain** [reply]
Et ? ca t'empeche d'écouter la commu ?

**Reply by BastLast** [reply]
non, juste la commu c'est pas vous 2 xD

**Reply by BastLast** [reply]
et ça me force pas à faire tout ce qui est proposé ici ou sur discord, ya un travail derriere et des choix à faire x)

**Reply by Voisin1234** [reply]
Meme sans les 4000 par semaine, vous êtes loin derrière même moi j'arrive à être dans le top cinq alors que je suis level 70

**Reply by Voisin1234** [reply]
Bon OK, pas cette semaine🤭 perso, j'ai dans un Chanel privé deux fois par jour, le classement score c'est assez rigolo de voir l'avancement (du moins l'avancement pour certains) et je peux montrer ( en screen ) tous les points qu'il y a eu depuis plus de trois mois visiblement c'est toujours les mêmes et ils n'ont pas forcément besoin du bonus league

**Reply by Carsebel** [reply]
@BananePlantain je l'espère. Regarde le tw en fin de semaine procion a souvent Raven aux fesses et la semaine dernière Evatco

**Reply by BananePlantain** [reply]
je sais Drapht mais là c'ets un débat et "de toute facon c'est moi le dev"n'est pas un argument pour cloturer

**Reply by BananePlantain** [reply]
donc Raven et Evatco ne peuvent pas espérer l'avoir sauf en comblant un écart immense (l'équivalent d'une journée de jeu pour certains)

**Reply by Ravenclaw-l** [reply]
une meilleure connaissance du jeu*

**Reply by Voisin1234** [reply]
![image](https://github.com/DraftBot-A-Discord-Adventure/DraftBot/assets/108148891/0d37e5f0-a6b4-45cb-a9ea-517ee5786a1f) Petit exemple, même sans le bonus ligue bah voilà voilà

**Reply by Ravenclaw-l** [reply]
Le screen de voisin prouve bien que sans le top gloire je n'aurais quand même pas gagné, puisque nous avons un écart de 2000 sur le screen et un écart de 1500 sur le bonus ligue.

**Comment by Gabrielre14** [comment]
Vous pensez pas qu'il serait plus pertinent de déplacer ce débat au chat général parce que là le faire via les commentaires Github c'est tout sauf pratique 🤣

**Reply by Voisin1234** [reply]
Ça a l'air de bien fonctionner pourtant 😇🤭

**Comment by Cirpacha** [comment]
C'est vrai qu'il y a un écart beaucoup trop grand entre les points donnés, le 2eme gagne 3200 donc si on n'a pas la première place dans le top gloire on n'a aucune chance pour le tw...

**Reply by BastLast** [reply]
> C'est vrai qu'il y a un écart beaucoup trop grand entre les points donnés, le 2eme gagne 3200 donc si on n'a pas la première place dans le top gloire on n'a aucune chance pour le tw...

Le but étant d'inviter les gens à fight

**Reply by Cirpacha** [reply]
Les inviter à fight pourquoi pas mais on pourrait aussi soit diminuer les points du premier, soit augmenter ceux des autres pour se rapprocher un peu, parce que là on est face à une impossibilité totale de gagner le tw si on n'a pas la 1ere place du top gloire (qui est quasi impossible à avoir aussi), c'est hyper décourageant quand on se consacre au bot 6 jours presque non-stop pour au final se faire laminer au dernier moment...

**Reply by Procion06** [reply]
mdr qu'est-ce qu'il ne faut pas lire xD
En gros le joueur qui gagne le top gloire gagne automatiquement le top week c'est ça ?
Quand, il n'y a pas si longtemps, Doc enchainait le top gloire, personne ne disait rien
Et en regardant tous les gagnants du tw lorsqu'il était top gloire, ça prouve juste que ton affirmation est totalement fausse...

**Reply by Cirpacha** [reply]
Déjà bonjour on se calme merci.
Je dis juste que si quelqu'un joue énormément et gagne le top gloire (comme c'est ton cas en ce moment), le tw n'est plus accessible à personne, et ça gâche tout son principe. Sans l'énorme influence du top gloire les autres auraient une chance, même minime, ce qui rendrait son utilité au tw, parce que là on ne peut vraiment rien faire.
J'ai tenté cette semaine, hier soir on avait à peu près le même nombre de points, aujourd'hui je me fais larguer c'est pas hyper agréable et surtout c'est irrémédiable, d'où mon soutien à la suggestion de banane.

**Comment by Procion06** [comment]
tu affirmes encore des choses fausses ça devient pénible
t'as gagné combien avec le bonus ligue ? 3000pts ? Environ 1000pts de moins par rapport à moi
j'ai presque 4000pts d'avance sur toi 
donc avec ou sans bonus ligue je suis large ^^

**Reply by Cirpacha** [reply]
Excuse-moi, tu peux arrêter de m'agresser et de tout rapporter à toi ?
Pour te répondre, j'ai aucun intérêt à farm pour le tw aujourd'hui vu que j'ai constaté ce matin la différence qui s'était creusée grâce au tw notamment, mais je pourrais la réduire un peu s'il m'en prenait l'envie. Si tu veux débattre de tes performances n'hésites pas à me contacter par messages privés mais c'est pas l'endroit.
Le fait est qu'hier on avait 50 points d'écart donc il est possible pour un joueur qui connaît suffisamment le jeu de t'égaler. Le soucis est dans les points donnés par le top gloire. C'est loin d'être une mauvaise idée d'en donner mais les écarts sont trop problématiques, même si j'étais 2eme j'aurais sûrement pas pu combler la différence créée. Et étant un être humain je me dois de dormir de temps en temps donc je ne peux pas jouer plus que ça, d'où le problème. Que ce soit toi ou un autre à la tête des tops, le tw doit rester accessible à plus d'une personne par semaine.

**Reply by Procion06** [reply]
je ne t'agresse en aucune façon, tu viens pleurer en affirmant plein de choses fausses, je te cite : "on est face à une impossibilité totale de gagner le tw si on n'a pas la 1ere place du top gloire" et je t'ai simplement démontré avec l'exemple de Doc que c'était faux
vous parlez de moi c'est légitime que je réponde et que je donne également mon point de vue, surtout quand tu dis nimp ^^
j'ai été le 1er à dire que le tw doit rester égalitaire pour tous et surtout les nouveaux, je vous invite à relire le post de la rage
maintenant Drapht en a décidé autrement et on fait avec 
1000 ou 1500pts d'avance sur une semaine grâce au bonus ligue c'est tout sauf impossible à rattraper

**Reply by Cirpacha** [reply]
Exact, on est face à une impossibilité de gagner et je le redis, si un joueur très actif est premier du top gloire il gagne le tw. Doc ne l'était pas autant que toi c'est tout. Mais au vu de tes victoires hebdomadaires il est juste plus que temps qu'on donne les faits. C'est super pour toi loulou, tu gagnes donc t'es content, mais nous on ne peut RIEN faire.

**Reply by Procion06** [reply]
Clairement non, suffit de louper la quoti de lieu tu perds 963pts et si ton concurrent est sur le chemin il a direct comblé l'avance de ton bonus ligue
je pourrai te donner plein d'exemples comme ça mais je pense que ça ne servirait à rien
"Mais au vu de tes victoires hebdomadaires il est juste plus que temps qu'on donne les faits"
sauf que j'enchainais déjà quelques tw avant la mise en place du bonus ligue
c'est confortable pour moi bien sûr mais arrêtez de dire que vous ne pouvez rien faire

**Reply by Cirpacha** [reply]
Mais en fait on n'a pas forcément envie d'attendre d'avoir une mission quotidienne ne de lieu et que monsieur procion la loupe, si qqn tente le tw et qu'il advient qu'il est meilleur que toi il mérite ce tw, c'est tout, je vois pas pourquoi un top indépendant et jamais réinitialisé entrerait dans les facteurs principaux de gains de points. Parce que dans ce cas autant tenir compte du classement global aussi, et pourquoi pas de celui de la guilde... Le top week est un top week. Il doit être représentatif de la performance des joueurs durant une semaine, pas de celle qui a été réalisée au préalable dans les combats. Il est impossible de devenir premier du top gloire en une semaine, donc les bonusligues ne devraient même pas inclure des points comptabilisés dans le tw. De plus même si tu arrêtais de fight cette semaine il est presque certain que tu gagnerais le top gloire, aucun mérite des 4k points. 
C'est juste logique, il faudrait soit que les points gagnés ne soient pas comptés dans le tw, soit réduire l'écart entre les points selon les positions du top, soit diminuer les points donnés.

**Reply by BastLast** [reply]
j'ai check, en supprimant le bonus ligue, procion aurait quand meme win cette semaine.

**Reply by BananePlantain** [reply]
oui mais quand tu vois quelqu'un qui est juste devant toi qui te met 4k d'un coup intérieurement tu pleures et t'abandonnes

**Reply by Cirpacha** [reply]
La question c'est pas de savoir s'il aurait été premier ou non, je vise pas procion, mais les points donnés ont rien à faire là.

**Comment by BananePlantain** [comment]
J'accuse personne,je donne mon avis (et je suis pas seul visiblement) et je l'illustre. Pourquoi faut que ca devienne une chasse aux sorcières/un concours d'égo ???

**Comment by BananePlantain** [comment]
https://discord.com/channels/429765017332613120/433538042901561365/1125772057427333121
CQFD

**Reply by Procion06** [reply]
ah mais je ne change pas d'avis, je viens de le dire plus haut ^^
sauf que moi je dis que la course est faussée, vous vous affirmez que c'est impossible xD

**Reply by BananePlantain** [reply]
faussé donc inéquitable donc impossible pour les nouveaux c'est la base de la discussion 😮‍💨

**Comment by BananePlantain** [comment]
Un samedi à 23h
![tw2](https://github.com/DraftBot-A-Discord-Adventure/DraftBot/assets/126412670/59fe7b68-185f-44a7-9ad9-5f25d4812769)
Le dimanche suivant à 12h
![tw1](https://github.com/DraftBot-A-Discord-Adventure/DraftBot/assets/126412670/52979f7f-b00d-42af-93a5-4e19898e689d)
Tiens c'est marrant ,les joueurs niveaux <80 ont quasiment tous disparues,comme par hasard. Je vous repose la question:considérer vous toujours que le topweek est équitable et accessible aux nouveaux joueurs ?

**Reply by romain22222** [reply]
je vais ptet dire une connerie, mais :
- seul le top 1 choppe un truc au topweek
- y'a tjrs autant de 80-, le top 7 ayant pas changé et les 4 suivants ont juste permutés, mais étant très proches, c'est normal

**Reply by BananePlantain** [reply]
tous le monde récuperent des points dans le bonusligue (et de l'argent,xp ...) le premier a en plus un badge

**Reply by Ravenclaw-l** [reply]
Tout ce que je vois sur tes captures, c'est que même sans le bonus ligue, les niveaux de -80 ne sont pas en tête du classement. Le bonus ligue n'a donc aucune incidence. C'est plutôt la connaissance du jeu qui compte.

**Reply by Carsebel** [reply]
Faudrait déjà que les screens soient plus d'actualité
![Screenshot_20240424_163139_Discord](https://github.com/DraftBot-A-Discord-Adventure/DraftBot/assets/120785212/9c18c41d-55e8-4f17-b29c-0dda8586810d)
![Screenshot_20240424_163228_Discord](https://github.com/DraftBot-A-Discord-Adventure/DraftBot/assets/120785212/658ab001-16cb-4854-8354-6adb683e9f7d)
Le top gloire c'est juste le fait que la personne fight et gagne. Mais faut pas oublier que c'est pas lui qui gagne le plus de points avec les fights.

**Reply by BananePlantain** [reply]
okay les screens date un peu c'est vrai . Hein bah si,si t'as plus de points tu gagnes ,je comprends pas ce te que veut dire en dessous des screens

**Reply by Voisin1234** [reply]
![image](https://github.com/DraftBot-A-Discord-Adventure/DraftBot/assets/108148891/93b9ae73-bdad-419f-9d75-e34646aab06c)!![image](https://github.com/DraftBot-A-Discord-Adventure/DraftBot/assets/108148891/99623631-d474-415d-9f41-d2a9b54d97c9)![image](https://github.com/DraftBot-A-Discord-Adventure/DraftBot/assets/108148891/e3515aff-48c6-41c8-95b1-f15f27eb47d0) Désolé, je n'ai pas le classement arrivé aux dernières minutes, mais avec ses screen plus récents, on pourra voir qu'ils sont loin derrière même sans bonus

**Reply by Carsebel** [reply]
Les points de gloire que l'on gagne sont en fonction de notre classement. 
Procion est 1er grâce à sa régularité à faire les fights durant la semaine. En etant  1er, il gagne moins de points de gloire que le reste du classement. En étant 2nde y a 3 semaine, j'ai gagné une quinzaine de points contre lui alors que moi je ne lui rapporte que 8 pts. Donc normal qu'il ai une compensation.

**Reply by Voisin1234** [reply]
Je ne saisis pas comment un joueur de faible niveau pourrait aisément atteindre le sommet hebdomadaire( sauf un dc) , par rapport à des vétérans qui maîtrisent depuis longtemps les mécaniques du jeu, ses subtilités et ses astuces. Pour ma part qu'il y ait un bonus ou autre, cela ne change rien au classement de la semaine

**Reply by romain22222** [reply]
de plus, je crois me souvenir que l'on gagne que 3 ou 4k points en étant 1er, donc avec 8k d'avance ca change absolument rien. En plus faut faire la différence avec ce que le second du TW gagne et qui aurait de toute facon été ignorée dans la vision populaire. Ce que je veux dire, c'est que le TW aurait un impact si quelqu'un était bien plus proche de lui dans le classement, là y'a juste pas photo

**Reply by romain22222** [reply]
raven l'a bien prouvé d'ailleurs la semaine passée

**Reply by Carsebel** [reply]
4k pour le 1er et le 2nde gangne 2900

**Reply by BastLast** [reply]
je maintient que ca me semble completement fair play que un mec qui sait jouer et qui joue régulièrement ai un avantage sur le tw.

**Reply by Carsebel** [reply]
3200 pts pour le 2nd pardon

**Reply by BastLast** [reply]
4000 points c'est clairement un gros avantage si tu veux gagner le tw mais ca empeche pas qqun qui a pas ce bonus de se classer dans le top 15, encore une fois perso le probleme il serait si yavais un mec lvl 15 qui gagnais le tw

**Reply by BastLast** [reply]
peut être que c'était pas comme ça avant mais les règes du jeu ont changé et c'est volontaire x)

**Comment by BastLast** [comment]
> je maintient que ca me semble completement fair play que un mec qui sait jouer et qui joue régulièrement ai un avantage sur le tw.

**Reply by BananePlantain** [reply]
Que son avantage soit sa connaissance du jeu,des events ok mais compensable par la régularité ,mais 
les combats n'ont rien à faire la dedans

**Reply by Ravenclaw-l** [reply]
Dans ce cas, on pourrait dire la même chose pour les missions difficiles réservées aux hauts levels, inaccessibles aux jeunes joueurs et qui rapportent plus de points. Les 4000pts s'ajoutent aux autres récompenses du bonus ligue et sont parfaitement legit.

**Reply by Carsebel** [reply]
Donc tu ne veux pas que le top gloire umpact le classement tw. Donc ça doit être appliqué à tous. 
Avant les pts de gloire, Procion etait déjà 1er donc tu veux faire quoi?

**Reply by BananePlantain** [reply]
oui les 4K score sont légits mais n'ont pas leurs places ici et rend le topweek inéquitable et inaccessible aux nouveaux (c'est la base de la discu je vous le rappelle)
MAIS RAAAAAAH VOUS RENDEZ FOU JE VISE PERSONNE
oui les points du bonusligue ne devrait pas compter dans le tw et ca pour tous,sans exceptions

**Reply by Ravenclaw-l** [reply]
Relis ton tout premier message stp

**Reply by BananePlantain** [reply]
oui ? y a écris "le badge doit etre accessible à tous" et je montre le cas de Procion,que je n'accuse de rien il est régulier c'est bien,mais ca montre que le bonusligue a trop d'impact

**Reply by Ravenclaw-l** [reply]
Tu as un exemple où un joueur de bas niveau a presque remporté la victoire, mais a perdu à cause du bonus ligue ?
Parce qu'on voit bien, semaine après semaine, que le classement reste stable malgré le bonus.

**Reply by Cyrpa** [reply]
Ben tout simplement le top week est un top réinitialisé chaque semaine, qui contient les points obtenus dans la semaine.
Si on y met des points obtenus grâce à une performance réalisée sur la durée, comme c'est le cas pour le top gloire qui n'est pas réinitialisé chaque semaine, ça le fausse.
Après, les points du top gloire sont mérités selon moi, pourquoi ne pas simplement les exclure du tw ? Chacun garderait ses points sur son profil, ils seraient comptés dans le top global mais pas dans le tw, qui deviendrait alors plus juste.

**Reply by BananePlantain** [reply]
je parle du concept général du tw tu me réponds exemples 🤔

**Reply by Procion06** [reply]
c'est malhonnête de ne parler que du top gloire dans ce cas-là , c'est dit juste au dessus, si tu veux être vraiment équitable faut aussi virer toutes les missions inaccessibles aux jeunes joueurs !
et pourquoi parler uniquement des 4K alors que tous les joueurs classés obtiennent un bonus ?
ou alors tu repars sur une chasse aux sorcières dont la légende dit que le tw ne serait accessible qu'à celui qui gagne le top gloire ? XD

**Reply by Ravenclaw-l** [reply]
Oui, tu soutiens que c'est impossible de gagner à cause du bonus, je te demande des cas concrets pour étayer ton affirmation 
¯\_(ツ)_/¯

**Reply by Cyrpa** [reply]
Possible de parler au nom de tout le monde plutôt que de se cracher dessus svp ? ça fait pas tellement avancer les choses

**Reply by BananePlantain** [reply]
Procion es ce que je parlee que des 4K ici ou pas dans : " oui les points du bonusligue ne devrait pas compter dans le tw et ca pour tous,sans exceptions" ?

**Reply by Ravenclaw-l** [reply]
Du coup, tu as des cas concrets où tes affirmations sont basées sur rien ?

**Reply by BananePlantain** [reply]
je peux défendre un concept ???
la chose que j'affirme est que le topweek est inaccessible aux nouveaux à cause du bonusligue c'est tout

**Reply by Procion06** [reply]
Banane tu en parles dès ton post d'ouverture : "Suffit de regarder Procion il domine le tw et paf 4k points gratuit qui le rend encore moins rattrapable"

**Reply by Ravenclaw-l** [reply]
Changer tout un système de récompenses sans même fournir de réels arguments. C'est brillant. Ne t'étonne pas si ça part un peu en vrille.

**Reply by Carsebel** [reply]
De toute façon, comment tu veux qu'un nouveau gagne le tw ? Il débute cherche la classe parfaite pour le jeu. Certains ne sont pas intéressés par le tw mais découvre le jeu avance dans les différentes missions.

**Reply by BananePlantain** [reply]
je me répete encore et encore,reprenons:tu es très actif donc tu as une avance +/- confortable ce que je critique est que ce bonus de points est inéquitable et fausse le tw

**Reply by Cyrpa** [reply]
en même temps je donne des arguments et vous ignorez, trop occupés à vous défoncer… non seulement c'est inutile mais en plus c'est débile.

**Reply by BananePlantain** [reply]
Y A DES ARGUMENTS 💢

**Reply by Ravenclaw-l** [reply]
Des arguments sans fondement. Tu affirmes que ça affecte le top alors que ça ne s'est jamais produit.

**Reply by BananePlantain** [reply]
c'est le principe d'un concept 🤦 donc non j'ai pas d'exemples mais un raisonnement qui se tient très bien "sans le bonusligue le tw serait plus équitable(si on met de coté l'inventaire,l'argent et la connaissance de jeu"

**Reply by Procion06** [reply]
je pense plutôt qu'on devrait augmenter le bonus ligue, genre à 10k points, comme ça on aurait des fights acharnés pour la 1ère place xD
et ça donnerait sa chance à un joueur motivé de gagner le tw !
parce que là, avec ou sans bonus, Raven et moi n'avons pas beaucoup de concurrence

**Reply by Ravenclaw-l** [reply]
"Le bonusligue à trop d'impact sur le tw ! "
est une affirmation, pas un concept. Tu confirmes quelque chose, tu n'exposes pas une idée.

**Reply by KirIcare** [reply]
Je te rassure, c'est pas avec plus de motivation que je gagnerais mes fights contre Ravenclaw xD
Je suis juste pas assez bon (pour l'isntant)

**Reply by BananePlantain** [reply]
alors Procion jamais de la vie ,y a des gens qui ne veulent pas combattre,les débutants ne peuvent pas aller chercher une 1e place dans un classement qui se réinitialise pas,je te dis que l'impact est déjà trop grand tu veux le rendre encore plus grand ???? .Raven,le titre est un affirmation et je défend le concept d'un tw accessible à tous
on tourne en rond 😒

**Reply by Ravenclaw-l** [reply]
Effectivement,  discussion stérile.
Encore une fois, enlève tous les points venant du bonus et le classement reste le même.

**Reply by Procion06** [reply]
c'était juste du troll, vu que ce thread est une vaste blague pour rester poli 
mon avis, qui n'a jamais changé, est que c'est inéquitable et je suis content de voir aujourd'hui des joueurs du même avis :)
maintenant Drapht a expliqué à maintes reprises pourquoi c'était comme ça mais tu reviens quand même à la charge

**Reply by BananePlantain** [reply]
Raven,non il reste pas le même du tout pour les nouveaux .Procion on est d'accord c'est inéquitable .Pourquoi je reviens à la charge : le débat n'est pas fini

**Comment by Carsebel** [comment]
Ça ne vise personne mais bon.
Quand d'autres occupaient le top, il n'y avait aucun problème.

**Reply by BananePlantain** [reply]
j'ai AUCUNS problèmes avec Procion,faut le dire en quelle langue ???

**Reply by Procion06** [reply]
pourtant dès ton 1er post tu me cites nommément, en spécifiant bien que t'en as marre de me voir dominer les tops
avec ce joli PS où tu dis que tu n'accuses personne de triche ^^

**Reply by BananePlantain** [reply]
je t'utilises en exemple pour montrer que le tw n'as besoin de ca pour avoir un écart déjà grand

**Reply by Procion06** [reply]
mais pourquoi rajouter en ps que tu n'accuses personne de triche, c'est quoi le rapport avec, je cite ton titre : "Le bonusligue a trop d'impact sur le tw !" ?

**Reply by BananePlantain** [reply]
je le précise parceque c'est comme ca qu'a tourné la discussion et que je vous l'avez interpreté

**Reply by Procion06** [reply]
pour reprendre tes termes, "comme par hasard", juste après ton post des joueurs se sont empressés de m'accuser de triche auprès du Staff (le même jour donc ^^)

**Reply by BananePlantain** [reply]
oui peut etre et mais je n'en fait pas parti

**Reply by Procion06** [reply]
pure coïncidence alors...

**Reply by Carsebel** [reply]
Mais cela servi de déclencheur

**Reply by BananePlantain** [reply]
mais j'y peut rien mdr je suis pas les autres

**Comment by Ntalcme** [comment]
Je propose ça comme ça mais pourquoi ne pas en faire un sondage simplement pour avoir l'avis de la communauté ?

**Reply by Ntalcme** [reply]
Et à partir de là soit l'on reste sur le système actuel soit l'on réfléchit à une solution communautaire.

**Reply by Ntalcme** [reply]
Je trouve ce débat trop long pour rien et aucun des partis ne semble vouloir lâcher le morceau alors ça me semble la solution la plus équitable.

**Reply by BastLast** [reply]
c'est mort.

**Reply by BastLast** [reply]
vous pouvez en discuter , j'ai tout lu, je vois juste des gens qui ralent et qui sont pas d'accord avec moi mais j'ai vu aucun argument pertinent pour changer le système autre que des gens qui aiment pas ça.

**Reply by BananePlantain** [reply]
j'adore m'embeter à proposer une idée ;exposer un raisonnement complète pour rien

**Reply by BastLast** [reply]
faut pas le prendre contre toi, je répond ca pour résumer grossierement tout le thread qui est parti un peu trop loin. Je vais faire une conclusion avec mon raisonnement et clore ce sujet définitivement.

**Reply by BananePlantain** [reply]
c'est parti loin est le minimum qu'on puisse dire 🤡

**Comment by aureochocob0n** [comment]
L'impact de ce débat aura été une gigantesque mascarade

**Comment by BastLast** [comment]
Du coup vu que tous ces threads partent un peu n'importe comment, je vais vous donner ma vision des choses et fermer cette fiche pour éviter que ça continue de partir en débat ici ( go continuer sur discord si vous n'êtes pas d'accord avec moi)

Le but du top week actuel est d'encourager les joueurs à mieux jouer à draftbot au sens large, et à récompenser la personne qui a gagné le plus de point dans la semaine.

Le but de ce classement n'a jamais été de récompenser les joueurs niveau 30 ou 50 qui sont des futures perles sur le jeu. Je comprends qu'avant, c'était un peu le cas, mais je ne vois pas du tout comme un problème que seuls des haut level aient une chance à remporter le tw

Pour moi les combats sont centraux au gameplay de draftbot, il est tout à fait logique qu'une personne qui y consacre beaucoup de temps ait un avantage conséquent en point. Je ne veux pas que cet aspect du gameplay soit oublié.
Le nombre de points gagnés a été assez bien étudié, sans être hyper bon, il me semble réalisable de gagner 2000 points au top gloire sans être un PGM, juste en faisant régulièrement des combats.

En bref, je ne considère pas ce sujet comme une priorité du tout, (on a du boulot sur la v5) mais il y aura tout de même des changements dans ce domaine avec la v5 (principalement parce que les combats tels qu'ils existent actuellement dans draftbot vont être refait complètement avec la v5 et que le nouveau système de combat sera probablement très différent du système actuel.

Conclusion, pour le moment ça reste comme ça, on verra ce qu'on fait pour la v5 je note vos avis, j'ai à peu près tout lu ici, j'en tiendrai compte en designant la suite des features de draftbot mais rien ne sera fait de ce que vous avez proposé. (mais ça veut pas dire que je vous écoute pas, juste que j'ai un plan qui va rendre toutes ces discussions obsolètes)

Bonne soirée
